{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -155,52 +244,167 @@ def generate_secret_key():
     )
 
 
-def create_structure():
-    """
-    Create the basic directory structure on the remote server.
-    """
-    run("mkdir -p %s" % env.root)
-
-    with cd(env.root):
-        run("mkdir -p static backups")
-        run("python3 -m venv venv")
+def is_supported_db_engine(engine):
+    return engine in [
+        "django.db.backends.postgresql_psycopg2",
+        "django.db.backends.postgresql",
+    ]
 
 
 @task
-def sync_settings():
+@remote
+def fetch_db(c, destination="."):
     """
-    Copy all settings defined in the environment to the server.
+    Dump the database on the remote host and retrieve it locally.
+
+    The destination parameter controls where the dump should be stored locally.
     """
-    for setting, value in env.settings.items():
-        set_setting(setting, value=value)
+    dump_path = c.conn.dump_db("~")
+    filename = os.path.basename(dump_path)
+
+    subprocess.run(
+        [
+            "scp",
+            "-P",
+            str(c.conn.port),
+            "{user}@{host}:{directory}".format(
+                user=c.conn.user, host=c.conn.host, directory=dump_path
+            ),
+            destination,
+        ]
+    )
+    c.conn.run("rm %s" % dump_path)
 
+    return os.path.join(destination, filename)
 
-def set_setting(setting_key, value=None, description=None):
+
+@task
+@remote
```

Comment:
Remove `@remote` decorator since `import_db` does not need a remote when `dump_file` is specified.